src/modules/AppHeader/components/AppHeader.js

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 import styled from 'styled-components';
 import theme from 'styles/theme';
 
-const Wrapper = styled.header`
-  align-items: center;
+const Wrapper = styled.div`
   background: ${theme.color.dark_gray};
   background-image: url(${theme.background.swirl});
   background-size: 4em;
-  display: flex;
-  padding: 1em;
+  display: grid;
+  grid-template-columns: 1fr minmax(auto, 600px) 1fr;
+  grid-template-areas:
+	  ". header-content .";
+  grid-area: header;
+  padding: 1em 0;
   margin: 0 0 2em;
+`;
+
+const Header = styled.header`
+  align-items: center;
+  display: flex;
+  grid-area: header-content;
```

Comment:
Wondering what your thoughts are on the benefits of using `grid` versus margin: `0 auto` and a `max-width` to center the content?

Reply:
It's a less-HTML way of making the header background span the entire width of the page. Grid (+ shiny toy fun) vs. extra wrapping `div`s. The tradeoff in negligible, but it was also a learning opportunity.

Reply:
(Plus I got to play around with `minmax()`, which isn't available outside of Grid.)